spec/cases/each_with_ar_sqlite.rb

```diff
@@ -29,11 +27,19 @@ class User < ActiveRecord::Base
 print "Parent: "
 puts User.first.name
 
+# https://www.sqlite.org/faq.html
+# Under Unix, you should not carry an open SQLite database across a fork() system call into the child process.
+ActiveRecord::Base.connection.disconnect!
+
 print "Parallel (#{in_worker_type}): "
 Parallel.each([1], in_worker_type => 1) do
+  ActiveRecord::Base.establish_connection(params)
```

Comment:
The reason I'm worried about breaking 0 with a given technique is that I can envision an application with optional Parallel use.  If we demonstrate a technique to use Parallel with SQLite, I feel like it should work both ways.

Personally I have my :test environment use 0 so my app which normally uses mysql can run it's specs with SQLite.